code/obj/table.dm

```diff
@@ -400,6 +411,35 @@
 	auto
 		auto = 1
 
+/obj/table/folding
+	name = "folding table"
+	desc = "A table with a faux wood top designed for quick assembly and toolless disassembly."
+	icon = 'icons/obj/table_folding.dmi'
+	parts_type = /obj/item/furniture_parts/table/folding
+
+	attack_hand(mob/user as mob)
+		if (user.is_hulk())
+			user.visible_message("<span style='color:red'>[user] collapses the [src] in one slam!</span>")
+			playsound(src.loc, "sound/impact_sounds/Generic_Hit_Heavy_1.ogg", 50, 1)
+			deconstruct()
+			src.set_density(0)
```

Comment:
What's the purpose of setting density here if it's just going to be qdeled? Also - I'm pretty sure deconstruct qdels it already, so this will runtime.

Reply:
I don't think it will runtime because that line will (hopefully) never be called after qdel because its a proc of the object being deleted, but it is redundant :P

Reply:
Did it that way because that's how it was handled in the existing code. Figured it was strange, but this means there's a runtime every time a hulk smashes a table in the main build. Fixed mine and the original.

Reply:
It should still call, since the object will be in nullspace. This calling proc would be a lingering reference to it, if I understand correctly